PULL_REQUEST_TEMPLATE

```diff
@@ -1,3 +1,17 @@
+<!--
+Uncomment this block and include PR # of change against 2.13.x where appropriate.
```

Comment:
Instructions in comment block changed per @emmberk's excellent suggestion.

Reply:
LGTM:+1: